agent/nmap_agent.py

```diff
@@ -41,6 +41,10 @@
 WIREGUARD_CONFIG_FILE_PATH = "/etc/wireguard/wg0.conf"
 DNS_RESOLV_CONFIG_PATH = "/etc/resolv.conf"
 
+DEFAULT_MASK_IPV6 = 128
```

Comment:
Does this fully address the issue. You mentioned some IP were coming with mask /64 from other agents.